services/galley/src/Galley/API/Internal.hs

```diff
@@ -117,11 +116,21 @@ internalAPI =
       <@> mkNamedAPI @"upsert-one2one" iUpsertOne2OneConversation
       <@> featureAPI
       <@> federationAPI
+      <@> conversationAPI
 
 federationAPI :: API IFederationAPI GalleyEffects
 federationAPI =
   mkNamedAPI @"get-federation-status" (const getFederationStatus)
 
+conversationAPI :: API IConversationAPI GalleyEffects
+conversationAPI =
+  mkNamedAPI @"conversation-channel" (const mempty)
```

Comment:
Such an absurd endpoint, I think it is not used. Let's just delete it?

Reply:
Done in d94b80b0a1734be55c1616f94201aaeaedf82ea2.